experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -96,6 +96,21 @@ describe("SharedTree", () => {
 		assert.deepEqual(toJsonableTree(view), [{ type: leaf.number.name, value: 5 }]);
 	});
 
+	it("handle in op", async () => {
+		const provider = await TestTreeProvider.create(2);
+		assert(provider.trees[0].isAttached());
+		assert(provider.trees[1].isAttached());
+
+		const field = provider.trees[0].editor.optionalField({
+			parent: undefined,
+			field: rootFieldKey,
+		});
+		field.set(
+			singleTextCursor({ type: leaf.handle.name, value: provider.trees[0].handle }),
+			true,
+		);
```

Comment:
Can you test blob handles? I think they are quite relevant for `SharedTree`. They are more likely to be used than the other handle types

Reply:
Blob handle?

Reply:
Or, rather, how does the handling of a blob handle differ from a DDS handle?

Reply:
Typing, the difference is typing. That's it

Reply:
Blob handles are of type `IFluidHandle<ArrayBufferLike>` which fails the type check of `IFluidHandle<FluidObject & IFluidLoadable>` as `ArrayBufferLike` does not have the property `IFluidLoadable`

Reply:
Is there an existing example of tests for blob handles? I only see e2e+gc tests that don't really test specific DDSes

Reply:
The only reason I recommend this testing for `SharedTree` is that likely the type of `IFluidHandle` stored in `SharedTree` will be `BlobHandles`. Other than that, I'm not particularly biased. Feel free to do this in a separate PR, I don't want to block this one.

Reply:
I don't think the runtime behavior is meaningfully different from the DDS perspective (it's not like any fluid handles we're testing today will be deref'd by SharedTree itself, just application code), but seems fine to add a separate test afterward for this. It's just a bit annoying as unit tests don't support the blob APIs so the set of things you need to mock is relatively large, at which point the test has a bit less value.